tests/basilisp/cli_test.py

```diff
@@ -185,7 +185,7 @@ def test_newline(self, run_cli):
 
     def test_simple_expression(self, run_cli):
         result = run_cli(["repl"], input="(+ 1 2)")
-        assert "basilisp.user=> 3\nbasilisp.user=> " == result.out
+        assert f"basilisp.user=> 3\nbasilisp.user=> " == result.out
```

Comment:
Does this need `\n` changed to `{os.linesep}` or should the `f` string be removed?

Reply:
Sorry this was unintentional, removed.